src/wallet/wallet.cpp

```diff
@@ -811,6 +812,23 @@ void CWallet::IncrementNoteWitnesses(const CBlockIndex* pindex,
                 }
             }
         }
+        // Sapling
+        for (uint32_t i = 0; i < tx.vShieldedOutput.size(); i++) {
+            const OutputDescription& outputDesc = tx.vShieldedOutput[i];
+            const uint256& note_commitment = outputDesc.cm;
```

Comment:
These two lines could be combined:

`const uint256& note_commitment = tx.vShieldedOutput[i].cm;`